blocks/editable/index.js

```diff
@@ -194,10 +194,41 @@ export default class Editable extends wp.element.Component {
 	}
 
 	onKeyDown( event ) {
+		const { keyCode } = event;
+		const moveUp = ( keyCode === UP || keyCode === LEFT ) && this.isStartOfEditor();
+		const moveDown = ( keyCode === DOWN || keyCode === RIGHT ) && this.isEndOfEditor();
+		const selectors = [
+			'*[contenteditable="true"]',
+			'*[tabindex]',
```

Comment:
Just found this library that looks interesting https://github.com/medialize/ally.js/blob/master/docs/api/README.md And it has a utility to find the tabbables elements https://github.com/medialize/ally.js/blob/master/docs/api/query/tabbable.md